docs/collection.md

```diff
@@ -113,12 +113,14 @@ $fruits->insert('banana', 1);
 
 $fruits->toArray(); // ['apple', 'banana', 'pear', 'peach']
 ```
+
 The insertion of an element, at a given position, moves the next elements of one position.
 
 !!! warning
-    If your *ArrayList* models an associative array (the indexes are strings) the insertion **overwrite** the element
+    If your *ArrayList* models an **associative array** (the indexes are strings) the insertion **overwrite** the element
```

Comment:
```suggestion
    If your *ArrayList* models an **associative array** (the indexes are strings) the insertion will **overwrite** the element
```

maybe the "will" can even be within the bold statement.

Reply:
Done.